tictactoeSpecific.md

```diff
@@ -0,0 +1,112 @@
+
+
+# TicTacToe Game Specification
+
+## About the Game:
+
+Tic-Tac-Toe, it's a game of two players, who take turns marking spaces (**X or O**), it can be any symbols)in a 3*3 grid.
+
+## Game Rules:
+
+As we said, it's a two-player game, where one player takes one symbol(**X**) and the second player takes one symbol(**O**).
```

Comment:
As we said is wrong. Remove that. Start with It's a Two player game. 

Also "one player takes one symbol" reads odd. Say "one player takes a Symbol/Marker". Marker will be more suitable.